GameServerLib/Content/NavGrid.cs

```diff
@@ -728,13 +728,13 @@ public static NavGrid ReadBinary(string filePath)
             return ReadData(b);
         }
 
-        public static NavGrid ReadBinary(byte[] fileBytes)
+        public static NavGrid ReadBinary(byte[] stream)
```

Comment:
Accidental change, but probably better if I didn't use byte[] to read the Navgrid.